src/nodewatcher/nodewatcher.py

```diff
@@ -223,7 +223,8 @@ def _wait_for_stack_ready(stack_name, region, proxy_config):
     stacks = cfn_client.describe_stacks(StackName=stack_name)
     stack_status = stacks["Stacks"][0]["StackStatus"]
     log.info("Stack %s is in status: %s", stack_name, stack_status)
-    return stack_status in ["CREATE_COMPLETE", "UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE"]
+    return stack_status in ["CREATE_COMPLETE", "UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE", "CREATE_FAILED",
+                            "UPDATE_FAILED"]
```

Comment:
can you remove the whitespace before `UPDATE_FAILED`

Reply:
To do so you can run `tox -e autoformat` like so:

```bash
pip install tox
cd cli
tox -e autoformat
```
Commit and push the changes! That'll make travis happy.

Reply:
Autoformating ran and branch updated.